### DISCUSSION TITLE
buddhism/2024/prajna_paramita/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#156] [General]
# buddhism/2024/prajna_paramita/

https://shanenull.com/buddhism/2024/prajna_paramita/

**Comment by shane0** [comment]
# wisdom goddess prajnaparamita

https://asia.si.edu/explore-art-culture/collections/search/edanmdm:fsg_S2015.26/

**Reply by shane0** [reply]
https://terebess.hu/english/Large-Wisdom-Conze.pdf

**Reply by shane0** [reply]
## 25k lines

- https://84000.co/translation/toh9

**Reply by shane0** [reply]
> “the contemplation of a bloated corpse, the contemplation of a worm-infested corpse, the contemplation of a putrefied corpse, the contemplation of a bloody corpse, {Dt.20} the contemplation of a blue-black corpse, the contemplation of a devoured corpse, the contemplation of a dismembered corpse, the contemplation of a skeleton, the contemplation of an immolated corpse, and the contemplation of the unpleasantness of food”
>> Excerpt From The Perfection of Wisdom in Twenty-Five Thousand Lines 84000 – Translating the Words of the Buddha

**Reply by shane0** [reply]
## 10k lines

- https://84000.co/translation/toh11

**Reply by shane0** [reply]
- https://en.wikipedia.org/wiki/Large_Praj%C3%B1%C4%81p%C4%81ramit%C4%81_S%C5%ABtras

**Comment by shane0** [comment]
# audio

- playlist https://youtube.com/playlist?list=PLJ1M78dFVbTyThaNlY0TFFAwB7cIMbccv&si=9qlrwfhXFPzizuaU

**Comment by shane0** [comment]
# emptiness

> Śāradvatīputra then asked, “Blessed Lord, how do bodhisattva great beings ‘engage’ with it when they engage with the perfection of wisdom?”
“Śāradvatīputra,” replied the Blessed One, “bodhisattva great beings are said to ‘engage’ with it when they engage with the emptiness of physical forms. {Dt.44} They are said to ‘engage’ with it when they engage with the emptiness of feelings, the emptiness of perceptions, the emptiness of formative predispositions, and the emptiness of consciousness. They are said to ‘engage’ with it when they engage with the emptiness of the eyes. They are said to ‘engage’ with it when they engage with the emptiness of the ears, the emptiness of the nose, the emptiness of the tongue, the emptiness of the body, and the emptiness of the mental faculty. They are said to ‘engage’ with it when they engage with the emptiness of sights. They are said to ‘engage’ with it when they engage with the emptiness of sounds, the emptiness of odors, [F.49.a] the emptiness of tastes, the emptiness of tangibles, and the emptiness of mental phenomena. They are said to ‘engage’ with it when they engage with the emptiness of the sensory element of the eyes, the emptiness of the sensory element of sights, and the emptiness of the sensory element of visual consciousness. They are said to ‘engage’ with it when they engage with the emptiness of the sensory element of the ears, the emptiness of the sensory element of sounds, and the emptiness of the sensory element of auditory consciousness; or with the emptiness of the sensory element of the nose, the emptiness of the sensory element of odors, and the emptiness of the sensory element of olfactory consciousness; or with the emptiness of the sensory element of the tongue, the emptiness of the sensory element of tastes, and the emptiness of the sensory element of gustatory consciousness; or with the emptiness of the sensory element of the body, the emptiness of the sensory element of touch, and the emptiness of the sensory element of tactile consciousness; or with the emptiness of the sensory element of the mental faculty, the emptiness of the sensory element of phenomena, and the emptiness of the sensory element of mental consciousness.
“They are said to ‘engage’ with it when they engage with emptiness of suffering, with the emptiness of the origin of suffering, with the emptiness of the cessation of suffering, and with the emptiness of the path.
“They are said to ‘engage’ with it when they engage with the emptiness of ignorance, with the emptiness of formative predispositions, with the emptiness of consciousness, with the emptiness of name and form, with the emptiness of the sense fields, with the emptiness of sensory contact, with the emptiness of sensation, with the emptiness of craving, with the emptiness of grasping, with the emptiness of the rebirth process, with the emptiness of actual birth, and with the emptiness of aging and death.
“They are said to ‘engage’ with it when they engage with the emptiness of all phenomena, as many as are designated as conditioned and unconditioned.
“Moreover, Śāradvatīputra, when bodhisattva great beings practice the perfection of wisdom, they are said to ‘engage’ with it when they engage with the emptiness of inherent existence. [F.49.b] So accordingly, Śāradvatīputra, when bodhisattva great beings practice the perfection of wisdom, they are said to ‘engage’ with it when they engage with those seven emptinesses.
“Those who engage with the perfection of wisdom through those seven emptinesses are not said to be engaged with it or not to be engaged with it. If you ask why, it is because they do not consider physical forms to be engaged with it or not engaged with it. They do not consider feelings, perceptions, formative predispositions, or consciousness to be engaged with it or not engaged with it.

**Reply by shane0** [reply]
> “They do not consider physical forms to move together with feelings. They do not consider feelings, perceptions, and formative predispositions to move together with consciousness. They do not consider consciousness to move together with formative predispositions. If you ask why, it is because there is no phenomenon that moves together with any [other] phenomenon. Owing to the emptiness of inherent existence, there is no moving, there is no departure, there is no meeting, and there is no association.
“Śāradvatīputra, the emptiness of physical forms is not physical forms. [F.50.a] The emptiness of feelings is not feelings. The emptiness of perceptions is not perceptions. The emptiness of formative predispositions is not formative predispositions. The emptiness of consciousness is not consciousness. Accordingly, the emptiness of physical forms does not obstruct, the emptiness of feelings does not feel, the emptiness of perceptions does not perceive, the emptiness of formative predispositions does not condition, {Dt.46} and the emptiness of consciousness does not cognize.
